infrastructure/modules/firewall-rule-collection-group/variables.tf

```diff
@@ -0,0 +1,70 @@
+variable "name" {
+  description = "The name of the firewall network rule."
+  type        = string
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9-]{1,80}$", var.name))
+    error_message = "The name must be a maximum of 80 characters and contain only alphanumeric characters."
+  }
+}
+
+variable "firewall_policy_id" {
```

Comment:
We've been putting all variables alphabetically (after name, resource group, location)